pkg/controller/build/build_controller_test.go

```diff
@@ -599,15 +429,14 @@ func testMCPIsDegraded(ctx context.Context, t *testing.T, cs *Clients) {
 
 	assertMachineConfigPoolReachesState(ctx, t, cs, "worker", func(mcp *mcfgv1.MachineConfigPool) bool {
 		// TODO: Should we fail the build without even starting it if the pool is degraded?
-		for _, condition := range getMachineConfigPoolBuildConditions() {
-			if apihelpers.IsMachineConfigPoolConditionTrue(mcp.Status.Conditions, condition) {
-				return false
-			}
-		}
+		// for _, condition := range getMachineConfigPoolBuildConditions() {
```

Comment:
suggestion (non-blocking): If this is commented out, we should probably just remove it.